### DISCUSSION TITLE
Using the character ‘@’ in an attribute name, crashes the linter tool.

### DISCUSSION THREAD

**Discussion by EPKLISN** [stoplightio/spectral#1688] [General]
The character ‘@’ in an attribute name, gives me the following output: “TypeError: Unknown value type…”.

Would it be possible to add the possibility to used that type of character ‘@’ in Spectral linter tool?

**Comment by P0lip** [comment]
Hey @EPKLISN.
Could you show me the ruleset and the document you use?

**Reply by EPKLISN** [reply]
OpenAPI version 3.0.x
Sorry, I can't show the API "as-is".
Still, what we have is an attribute, any name, and we add "@" as the first character. E.g. <@bob> (exclude < and >). The linter tool crashed (js dump), so I added <'@bob'> (exclude < and >) as a string definition, and it still crashed (js dump).
Is it the character '@' itself that gives me the issue, or how should I do it?

**Reply by P0lip** [reply]
Hey!
My apologies for such a delayed response. It fell through the cracks.
Could you give it a try now? I think the new version, namely v6, should handle it just fine.

**Reply by EPKLISN** [reply]
It does now, thanks.
